src/middlewared/middlewared/plugins/apps/rollback.py

```diff
@@ -81,6 +92,29 @@ def rollback(self, job, app_name, options):
                         }
                     )
 
+            if options['rollback_hostpath_snapshots']:
+                if host_path_mapping:
+                    logger.debug('Rolling back hostpath snapshots for %r app', app_name)
+
+                for host_path, dataset in host_path_mapping.items():
```

Comment:
Are we able to differentiate between host paths that contain user data vs. config data for the app? I can maybe understand rolling back latter, but not former.